### DISCUSSION TITLE
One of the first uses of uranium was as a coloring agent for what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#143] [Galaxy Trivia] [answered]
**Hint:** 🏺

**Comment by Tynab** [comment, marked as answer]
One of the first uses of uranium was as a coloring agent for glass and ceramics. The addition of uranium oxide to glass or glaze produces a yellow or greenish color, depending on the concentration of the uranium. This property was first discovered in the early 19th century, and uranium glass became popular in the late 19th and early 20th centuries for decorative items such as vases, plates, and glassware. However, the use of uranium in this way declined in the mid-20th century due to concerns about its radioactivity. Today, uranium is primarily used as a fuel for nuclear power plants and in the production of nuclear weapons.

**Reply by ParanoidUser** [reply]
Congrats, you made it up to the gold! 🥇
